File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        int len= nums.size();
       for (int i = 0; i < len-1; i++)
       {
        for (int j = i+1; j<len; j++)
        {
            if (nums[i] == nums[j]) 
            {
                return true;
            }
        }
       } 
       return false;
    }
};

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        
        int length = nums.length;
        int[] suffix = new int[length];
        int[] prefex = new int[length];
        int[] output = new int[length];

        prefex[0] = 1;
        suffix[0] = 1;
        for (int i = 1; i < length; i++) {
            prefex[i] = prefex[i-1]*nums[i-1];
            suffix[i] = suffix[i-1]*nums[length-i];
        }

        for (int i = 0; i < length; i++)
            output[i] = prefex[i]*suffix[length-i-1];

        return output;
    }
}  
